ReviewBot.py

```diff
@@ -169,7 +169,10 @@ def request_override_check_users(self, project):
         self.staging_api(project)
         config = self.staging_config[project]
 
-        users = group_members(self.apiurl, config['staging-group'])
+        users = []
```

Comment:
Actually, since this can now return `len(users) = 0` for non staging bots a condition could be added to skip override check since no comments will be allowed anyway. Alternatively project maintainers could be used instead.